README.md

```diff
@@ -1,102 +1,220 @@
 # terraform-azurerm-aks
+
 ## Deploys a Kubernetes cluster on AKS with monitoring support through Azure Log Analytics
 
 This Terraform module deploys a Kubernetes cluster on Azure using AKS (Azure Kubernetes Service) and adds support for monitoring with Log Analytics.
 
 -> **NOTE:** If you have not assigned `client_id` or `client_secret`, A `SystemAssigned` identity will be created.
 
-## Usage in Terraform 0.13
+## Notice on Upgrade to V5.x
+
+V5.0.0 is a major version upgrade and a lot of breaking changes have been introduced. Extremely cautious must be taken during the upgrade to avoid resource replacement and downtime by accident.
```

Comment:
```suggestion
V5.0.0 is a major version upgrade and a lot of breaking changes have been introduced. Extreme caution must be taken during the upgrade to avoid resource replacement and downtime by accident.
```